plugins/hls-class-plugin/hls-class-plugin.cabal

```diff
@@ -28,6 +28,7 @@ library
     , base            >=4.12 && <5
     , containers
     , ghc
+    -- Require ghc-exactprint >= 1.5 on ghc9.2
```

Comment:
> oops, I think we can only add a constraint in .project or stack-x.yaml since ghc-exactprint-0.6 is required from ghc8 to ghc9.0. I'll add a comment here.

Unfuntunately, just a comment.

Reply:
You could have a cabal conditional based on the GHC version? Maybe it's not worth it.